### PR TITLE
feat: add sign-in page

### PR DESCRIPTION
**Pull request by InbarDanieli**

### CODE REVIEW

File: pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import NextAuth from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+console.log(process.env.NEXTAUTH_URL);
```
Comment (OmriBarZik) [inline pages/api/auth/[...nextauth].ts:3]: @InbarDanieli please remove log

File: pages/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { GetServerSideProps } from "next";
+import { useRouter } from "next/router";
+import { useSession, signIn, getProviders } from "next-auth/react";
+
+export default function SignIn({ providers }: any) {
+  const router = useRouter();
+  const { data: session } = useSession();
+  session && session.user && router.replace("/");
```
Comment (OmriBarZik) [inline pages/login/index.tsx:8]: @InbarDanieli  please make it an if statement like so

```ts
if (session && session.user) router.replace("/");
```

**Review by OmriBarZik — CHANGES_REQUESTED** [review]

File: pages/api/auth/[...nextauth].ts (hunk 1 above)
Comment (InbarDanieli) [reply pages/api/auth/[...nextauth].ts]: done

File: pages/login/index.tsx (hunk 2 above)
Comment (InbarDanieli) [reply pages/login/index.tsx]: done

**Review by OmriBarZik — APPROVED** [review]

**Pull request merged**